scripts/report_scripts/index.Rmd

```diff
@@ -136,6 +136,11 @@ plot_ly (data = melted,
                            style = 'open-street-map'))
 ```
 
+## Raw data VOC frequencies per sample
+```{r, include = F}
+variant_freqs <- readLines(params$variants_csv) %>% paste0(collapse="\n") %>% base64_urlencode()
+``` 
+[Download Variant_frequencies.csv](`r sprintf('data:text/csv;base64,%s', variant_freqs)`)
```

Comment:
Please squash the commit that adds the base64url library into this commit.  We want the pipeline to be runnable at every commit.  This commit here introduces a bug, which can be fixed by squashing it with the commit adding the `library(...)` line.